atd-toolbox/data_model/python/populate_edits_sql/people.sql

```diff
@@ -0,0 +1,193 @@
+set client_min_messages to warning;
+
+-- remove all names from old people tables for non-fatalities
+-- this can take more than a minute per table
+update atd_txdot_person set prsn_last_name = null, prsn_first_name = null, prsn_mid_name = null
+where prsn_injry_sev_id != 4;
+update atd_txdot_primaryperson set prsn_last_name = null, prsn_first_name = null, prsn_mid_name = null
+where prsn_injry_sev_id != 4;
+
+-- double check if cris has the right data for this person records
+update atd_txdot_primaryperson set prsn_injry_sev_id = 0 where prsn_injry_sev_id = 94;
```

Comment:
wait whats goin on here?

Reply:
ah, sorry. that comment is actually a todo to see if records in the old schema with `prsn_injry_sev_id` = 94 actually have an updated value in more recent CRIS extracts, because we've never processed CRIS updates on person records in the old DB. 

Basically I want to check my assumption that it's ok  to set  `94` (reported invalid) to `0` (unknown).